src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowTrimming.Fields.Edit.cshtml

```diff
@@ -0,0 +1,111 @@
+@using OrchardCore.Workflows.Models
+@using OrchardCore.Workflows.Trimming.Services
+@using OrchardCore.Workflows.Trimming.ViewModels
+@model WorkflowTrimmingViewModel
+
+<div class="form-check">
+    <input type="checkbox" class="form-check-input" asp-for="Disabled" checked="@Model.Disabled" />
+    <label class="form-check-label" asp-for="Disabled">@T["Disable"]</label>
+    <span class="hint dashed">@T["Whether the task is disabled."]</span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="RetentionDays">
+    <label asp-for="RetentionDays" class="control-label">@T["Retention period"]</label>
+    <input asp-for="RetentionDays" type="number" class="form-control" />
+    <span class="hint">@T["The number of days a workflow instance is retained."]</span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="LastRunUtc">
+    <label asp-for="LastRunUtc" class="control-label">@T["Last run"]</label>
+    <div class="form-control">
+        @(Model.LastRunUtc.HasValue
+            ? await DisplayAsync(await New.DateTime(Utc: Model.LastRunUtc.Value, Format: "g"))
+            : T["Never"])
+    </div>
+    <span class="hint">@T["The last time the background trimming task was run."]</span>
+</div>
+
+@* We use a hard-coded list of statuses here instead of iterating over the enum so the explicit T-strings make
+       localization easier (and so the default English displayed string can be different than the enum value's name). *@
```

Comment:
```suggestion
@* We use a hard-coded list of statuses here instead of iterating over the enum to simplify localization with explicit T-strings, allowing the default English display text to differ from the enum value names. *@
```

Reply:
This would produce a really long, 212 character long line, making it inconvenient to work with.